sig-autoscaling/charter.md

```diff
@@ -0,0 +1,78 @@
+# SIG Autoscaling Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses
+the Roles and Organization Management outlined in [sig-governance].
+
+## Scope
+
+Covers development and maintenance of components for automated scaling in
+Kubernetes.  This includes automated vertical and horizontal pod
+autoscaling, initial resource estimation, cluster-proportional system
+component autoscaling, and autoscaling of Kubernetes clusters themselves.
+
+### In scope
+
+- Autoscaling-related API objects, such as the HorizontalPodAutoscaler and
+  VerticalPodAutoscaler
+
+- Autoscaling-related tools, such as the cluster autoscaler,
+  single-component scaling tools (e.g. pod-nanny), and
+  cluster-proportional scaling tools
+
+- Ensuring API interfaces (the scale subresource) are available and usable
+  to enable other SIG to write autoscalable objects, and enable people to
+  interact with those interfaces.
+
+[Link to SIG section in sigs.yaml][sigs.yaml]
+
+#### Code, Binaries and Services
+
+- Components and utilities that take automated action to scale a component
+  on the cluster
+
+- Components and utilities that take automated action to scale the cluster
+  itself
+
+- Special parts of client-go for interacting with with the scaling
```

Comment:
Definitely need a pointer into the project here.

Reply:
done -- mentioned the specific subproject.  Actual folder definitions are in sigs.yaml